snowfakery/data_generator_runtime_dom.py

```diff
@@ -33,14 +33,19 @@ class FieldDefinition(ABC):
          fieldname: X
     """
 
-    def __init__(self):
-        self.definition = None
-        self.filename = None
-        self.line_num = None
-
     @abstractmethod
     def render(self, context: RuntimeContext) -> FieldValue:
-        pass
+        pass  # noQA
```

Comment:
Were you trying to make coverage.py skip this? Because this isn't the right invocation for that. I'd suggest configuring coverage.py to ignore abstract methods like we have for CumulusCI: https://github.com/SFDO-Tooling/CumulusCI/blob/master/.coveragerc#L8

Reply:
Fixed in 9632295 and 85112bf